media-libs/ilmbase/ilmbase-2.5.4.ebuild

```diff
@@ -29,17 +38,17 @@ multilib_src_configure() {
 	)
 
 	# Disable use of ucontext.h wrt #482890
-	if use hppa || use ppc || use ppc64; then
-		mycmakeargs+=(
-			-DILMBASE_HAVE_UCONTEXT_H=OFF
-		)
-	fi
+#	if use hppa || use ppc || use ppc64; then
```

Comment:
Hey, what're all these comments about?

Reply:
I have neither of these machines available to test whether these are still needed. Should have been fixed according to the cmake file, but I couldn't test it.